posts/2023-04-04-23.0.0.3.adoc

```diff
@@ -0,0 +1,600 @@
+
+[[cdi]]
+|https://jakarta.ee/specifications/annotations/1.3/[Dependency Injection 2.0]
+|Unchanged
+|link:{url-prefix}/docs/latest/reference/feature/cdi-4.0.html[cdi-4.0]
+
+|===
+
+NOTE: Jakarta EE Core Profile 10 also introduces CDI Lite 4.0, which is a subset of the features provided by CDI 4.0 and is therefore made available as part of the link:{url-prefix}/docs/latest/reference/feature/cdi-4.0.html[cdi-4.0] feature.
+
+[#webprofile]
+== Jakarta EE Web Profile 10
+[cols="4,2,3"]
+|===
+|Specification |Updates |Liberty Feature Documentation
+
+|https://jakarta.ee/specifications/coreprofile/10/[Jakarta EE 10 Core Profile]
+|New
+|<<coreprofile, See previous table>>
+
+|https://jakarta.ee/specifications/authentication/3.0/[Authentication 3.0]
+|Major update
+|link:{url-prefix}/docs/latest/reference/feature/appAuthentication-3.0.html[appAuthentication-3.0]
+
+|https://jakarta.ee/specifications/cdi/4.0/[Context Dependency Injection 4.0]
+|Major update
+|link:{url-prefix}/docs/latest/reference/feature/cdi-4.0.html[cdi-4.0]
+
+|https://jakarta.ee/specifications/concurrency/3.0/[Concurrency 3.0]
+|Major update
+|link:{url-prefix}/docs/latest/reference/feature/concurrent-3.0.html[concurrent-3.0]
+
+|https://jakarta.ee/specifications/expression-language/5.0/[Expression Language 5.0]
+|Major update
+|link:{url-prefix}/docs/latest/reference/feature/expressionLanguage-5.0.html[expressionLanguage-5.0]
+
+|https://jakarta.ee/specifications/faces/4.0/[Expression Language 5.0]
+|Major update
+|link:{url-prefix}/docs/latest/reference/feature/faces-4.0.html[faces-4.0]
+
+|https://jakarta.ee/specifications/security/3.0/[Security 3.0]
+|Major update
+|link:{url-prefix}/docs/latest/reference/feature/appSecurity-5.0.html[appSecurity-5.0]
+
+|https://jakarta.ee/specifications/servlet/6.0/[Servlet 6.0]
+|Major update
+|link:{url-prefix}/docs/latest/reference/feature/servlet-6.0.html[servlet-6.0]
+
+|https://jakarta.ee/specifications/tags/3.0/[Standard Tag Library 3.0]
+|Major update
+|link:{url-prefix}/docs/latest/reference/feature/servlet-6.0.html[pages-3.1]
+
+|https://jakarta.ee/specifications/persistence/3.1/[Persistence 3.1]
+|Minor update
+|link:{url-prefix}/docs/latest/reference/feature/persistence-3.1.html[persistence-3.1]
+
+|https://jakarta.ee/specifications/pages/3.1/[Server Pages 3.1]
+|Minor update
+|link:{url-prefix}/docs/latest/reference/feature/pages-3.1.html[pages-3.1]
+
+|https://jakarta.ee/specifications/websocket/2.1/[WebSocket 2.1]
+|Minor update
+|link:{url-prefix}/docs/latest/reference/feature/websocket-2.1.html[websocket-2.1]
+
+|https://jakarta.ee/specifications/bean-validation/3.0/[Bean Validation 3.0]
+|No change
+|link:{url-prefix}/docs/latest/reference/feature/beanValidation-3.0.html[beanValidation-3.0]
+
+|https://jakarta.ee/specifications/debugging/2.0/[Debugging Support for Other Languages 2.0]
+|No change
+|Not applicable
+
+|https://jakarta.ee/specifications/enterprise-beans/4.0/[Enterprise Beans Lite 4.0]

```

Comment:
```suggestion
|https://jakarta.ee/specifications/enterprise-beans/4.0/[Enterprise Beans 4.0 Lite]

```

Reply:
I've seen this one referenced both ways, but the umbrella spec does reference it with Lite at the end so I'll change it to that